guidelines/static-websites.md

```diff
@@ -0,0 +1,137 @@
+# Rust Infrastructure hosting for static websites
+
+The Rust Infrastructure team provides hosting for static websites available for
+all Rust teams. This document explains the requirements a website needs to meet
+and how to setup one.
+
+## Requirements for hosting websites
+
+* **The website must be managed by a Rust team, or be officially affiliated with
+  the project.**  
+  The infrastructure team has finite resources and we can't offer hosting for
+  community projects.
+* **The website’s content and build tooling must be hosted on a GitHub
+  repository in either the [rust-lang](https://github.com/rust-lang) or
+  [rust-lang-nursery](https://github.com/rust-lang-nursery) organizations.**  
+  The infrastructure team must be able to rebuild the website content at any
+  time (for example if we need to switch hosting), and having it hosted on a
+  GitHub repository inside infra-managed organizations is the best way for us
+  to ensure that. Even though we'd prefer for all the repositories to be public
+  it's not a requirement.
+* **The website must be built and deployed with Travis CI.**  
```

Comment:
Reworded the section.

Reply:
I'd probably personally change this to just "CI", I don't think adaptation to other toolings will be that hard (and almost all CI services are the same in terms of the feature set we'd need for site deployment here)